content/200-orm/100-prisma-schema/20-data-model/90-multiple-files.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Multi-file Prisma schema
+metaTitle: Prisma schema with multiple files
+metaDescription: Learn how to split your Prisma schema into multiple files.
+---
+
+:::tip
+
+Multi-file Prisma schema is available via the `prismaSchemaFolder` preview feature in Prisma versions 5.15.0 and later.
+
+:::
+
+## How to enable multi-file Prisma schema support
+
+Multiple Prisma schema file support is currently in preview. To enable the feature, add the `prismaSchemaFolder` feature flag to the `previewFeatures` field of the `generator` block in your Prisma schema: 
+
+```prisma file=schema.prisma showLineNumbers
+generator client {
+  provider        = "prisma-client-js"
+  //add-next-line
+  previewFeatures = ["prismaSchemaFolder"]
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+```
+
+## How to include multiple Prisma schema files in your project
+
+To use multiple Prisma schema files, add a `schema` inside of your current `prisma` directory. With the `prismaSchemaFolder` Preview feature enabled, you can add as many files as you want to the `prisma/schema` directory.
+
+## How to use existing Prisma CLI commands with multiple Prisma schema files
+
+Existing Prisma CLI commands have been updated to support multiple Prisma files. Simply replace references to a file with a directory. As an example, the following `prisma migrate diff` command:
```

Comment:
Before this, we could mention that any command that reads the Prisma Schema under the hood _will just work_ and pick out correctly if you have a file or a folder, and then load all the files in that folder to combine them.

Only if a CLI command has an explicit option to provide a schema - it was file before, but now also accepts folder names. `--schema` is the universal option, so something like `db push --schema ...` is probably a better example. 

(`migrate diff` is actually an outlier)